add json schema for substreams manifest (#155)
streamingfast/substreams: docs/developers-guide/creating-your-manifest.md

## Manifest walkthrough

{% hint style="success" %}

> **Tip**: When writing and checking your `substreams.yaml` file, it may help to check your manifest against our [JSON schema](https://json-schema.org/) to ensure there are no problems. JSON schemas can be used in [Jetbrains](https://www.jetbrains.com/help/idea/json.html#ws_json_schema_add_custom) and [VSCode](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml). Our manifest schema is located at `/schemas/manifest-schema.json`.

Reviewer: Use proper linking directly to schema file:

```suggestion
**Tip**: When writing and checking your `substreams.yaml` file, it may help to check your manifest against our [JSON schema](https://json-schema.org/) to ensure there are no problems. JSON schemas can be used in [Jetbrains](https://www.jetbrains.com/help/idea/json.html#ws_json_schema_add_custom) and [VSCode](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml). Our manifest schema can be seen [here](../../schemas/manifest-schema.json).
```

Adjust the `../../schemas/manifest-schema.json` accordingly. I'm unsure if Git books will render it with the correct link because it's outside of `docs` folder. We could check once merged and adjust using an absolute GitHub link if it doesn't work properly.

Author: sounds good
